src/plone/app/transmogrifier/uidupdater.py

```diff
@@ -59,6 +64,12 @@ def __iter__(self):
                         obj._setUID(uid)
 
             if IAttributeUUID.providedBy(obj):
+                current_uid = IUUID(obj.aq_base, None)
+                existing_uid = obj.portal_catalog({'UID': uid})
+                if bool(existing_uid):
+                    logger.debug("Existing UID {} on {}, this item: {}".format(
```

Comment:
this is not a warning, you should use `logger.debug()` instead; also, don't use `{}` as we will lose compatibility with Python < 2.7, IIRC.